### ISSUE TITLE
Gradient descent algorithm and learning rate

### ISSUE THREAD

**Issue by skyer9** [hunkim/DeepLearningZeroToAll#129]
I think 'learning rate' should be explained with 'Gradient descent algorithm'.

```python
import numpy as np
from keras import optimizers
from keras.layers import Dense
from keras.models import Sequential


x_train = [1, 2, 3, 4]
y_train = [0, -1, -2, -3]

model = Sequential()
model.add(Dense(1, input_dim=1))

sgd = optimizers.SGD(lr=0.1)
model.compile(loss='mse', optimizer=sgd)

# prints summary of the model to the terminal
model.summary()

model.fit(x_train, y_train, epochs=100)

y_predict = model.predict(np.array([5]))
print(y_predict)
```

this code trains only 100 epochs and more precise.

**Comment by kkweon** [comment]
I don't understand this.
Can you elaborate on your issue?
And which file are you referring to?

**Comment by skyer9** [comment]
https://github.com/hunkim/DeepLearningZeroToAll/blob/master/Keras/klab-02-1-linear_regression.py

original source code uses default value for learning rate.
it makes slows down learning time 10x.

**Comment by kkweon** [comment]
Aha, feel free to send us a PR
